instrumentation/rxjava-2.0/javaagent/src/main/java/io/opentelemetry/instrumentation/rxjava2/RxJava2InstrumentationModule.java

```diff
@@ -41,15 +42,16 @@ public RxJava2InstrumentationModule() {
     @Override
     public Map<? extends ElementMatcher<? super MethodDescription>, String> transformers() {
       return Collections.singletonMap(
-          isTypeInitializer(),
-          RxJava2InstrumentationModule.class.getName() + "$RxJavaPluginsAdvice");
+          isMethod(), RxJava2InstrumentationModule.class.getName() + "$RxJavaPluginsAdvice");
     }
   }
 
   public static class RxJavaPluginsAdvice {
-    @Advice.OnMethodExit(suppress = Throwable.class)
-    public static void postStaticInitializer() {
-      TracingAssembly.enable();
+
+    // TODO(anuraaga): Replace with adding a type initializer to RxJavaPlugins
```

Comment:
@trask Perhaps we can merge and fix forward if the type initializer stuff is too complicated right now.

Reply:
Do we have a registered issue for this? If yes, can you reference it here?

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/2685, and pushed to add to the comment